etc/lime-elements.api.md

```diff
@@ -1551,6 +1563,15 @@ namespace JSX_2 {
         "onChangeTab"?: (event: LimelTabPanelCustomEvent<Tab>) => void;
         "tabs"?: Tab[];
     }
+    // @beta
+    interface LimelTextEditor {
+        "disabled"?: boolean;
+        "helperText"?: string;

```

Comment:
Talked to Lucy, at present `helpertext` has been confused with `placeholder`

Reply:
Wherever it says `helpertext` read `placeholder` instead 😅

Reply:
not anymore. i changed them all